client/app/bundles/Zoon/components/WorkflowCreator.jsx

```diff
@@ -187,6 +188,21 @@ class WorkflowCreator extends React.Component {
     return this.families().length
   }
 
+  showModuleLightbox = (moduleId) => {
+    if (moduleId === null || moduleId === undefined) {
```

Comment:
Hmm. That one was out of generalized fear of undefineds.

re reference: Derp, tried to pull it back up, but I can't seem to find it. I do remember it was a rule in a respectable linter (JSHint? JSLint? One of those older ones).

Given the way I'm handwaving all of this, I can't defend my position for crispness (since it's being crisp about nothing too set, anyway). Given none of the [falsy values](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) are useful, I think I'm just gonna do that.